packages/react-styled-core/src/Spinner/index.js

```diff
@@ -0,0 +1,114 @@
+import { forwardRef } from 'react';
+import { keyframes } from '@emotion/core';
+import useTheme from '../useTheme';
+import Box from '../Box';
+import PseudoBox from '../PseudoBox';
+
+const rotate = keyframes`
+  100% {
+    transform: rotate(360deg);
+  }
+`;
+
+const dash = keyframes`
+  0% {
+    stroke-dasharray: 1, 200;
+    stroke-dashoffset: 0;
+  }
+  50% {
+    stroke-dasharray: 89, 200;
+    stroke-dashoffset: -35px;
+  }
+  100% {
+    stroke-dasharray: 89, 200;
+    stroke-dashoffset: -124px;
+  }
+`;
+
+const sizes = {
+  xs: {
+    width: 16,
+    strokeWidth: 2,
+  },
+  sm: {
+    width: 32,
+    strokeWidth: 2,
+  },
+  md: {
+    width: 48,
+    strokeWidth: 2,
+  },
+  lg: {
+    width: 64,
+    strokeWidth: 4,
+  },
+  xl: {
+    width: 80,
+    strokeWidth: 4,
+  },
+};
+
+const Spinner = forwardRef(
+  (
+    {
+      size = 'md',
+      label = 'Loading...',
+      color,
+      ...props
+    },
+    ref,
+  ) => {
+    const { colors } = useTheme();
+    const _width = sizes[size] ? sizes[size].width : sizes.md.width;
+    const _strokeWidth = sizes[size] ? sizes[size].strokeWidth : sizes.md.strokeWidth;
+    const _strokeColor = color ? (colors[color] || color) : colors['blue:60'];
```

Comment:
Instead of calling the `useTheme` hook to get the `colors` object, this line can be simplified as below 
```jsx
const defaultColor = 'blue:60';
const _strokeColor = color ?? defaultColor;
```